packages/core/src/prompts/lesson-assistant/parts/currentLessonPlan.ts

```diff
@@ -0,0 +1,10 @@
+import { TemplateProps } from "..";
+
+export const currentLessonPlan = ({
+  currentLessonPlan,
+}: TemplateProps) => `THE CURRENT LESSON PLAN
+This is where we have got to with the lesson plan so far.
+This represents the state of the application at the moment, as a JSON document.
+Use this as the basis for making any changes to the lesson plan.
+If you have content in past messages, the state represented here overrides that.
+${currentLessonPlan}`;
```

Comment:
Was about to say should this be JSON.stringified but just looked and it's expected as a string